<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyStudentClass</class>
 <widget class="QMainWindow" name="ModifyStudentClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModifyStudent</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="snoLine1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="okBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="snoLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLine">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sexBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>女</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sdeptBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>信息科学于技术学院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>心理学院</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
